services/compute/virtualmachinescaleset/virtualmachinescaleset.go

```diff
@@ -422,6 +444,14 @@ func (c *client) getWssdVirtualMachineScaleSetSecurityConfiguration(vmp *compute
 	return wssdsecurity, nil
 }
 
+func (c *client) getWssdVirtualMachineScaleSetGuestAgentConfiguration(vmp *compute.VirtualMachineScaleSetVMProfile) (*wssdcloudcompute.GuestAgentConfiguration, error) {
```

Comment:
same here

Reply:
The other functions also have the error as a returned value, so we added it too for consistency.